pkg/actuators/machine/reconciler.go

```diff
@@ -211,6 +211,13 @@ func (r *Reconciler) update() error {
 	runningLen := len(runningInstances)
 	var newestInstance *ec2.Instance
 
+	//Prepare the tag list with infrastructure tags.
+	//these tags will update to the ec2 instance.
```

Comment:
Nit, we tend to have a space before the text in comments across our projects, also, a suggestion for a rewording
```suggestion
	// Prepare the tag list with infrastructure tags.
	// These tags will be used to update the EC2 instance tags.
```

Reply:
Sure, will take now & further :)